src/amo/utils/addonExtract.ts

```diff
@@ -2,19 +2,23 @@ import * as extract from "extract-zip";
 import * as fs from "fs";
 import * as vscode from "vscode";
 
+function dirExistsOrMake(dir: string) {
+  if (!fs.existsSync(dir)) {
+    fs.mkdirSync(dir);
+    return false;
+  } else {
+    return true;
+  }
```

Comment:
only a suggestion, but imo this works better if it returns `true` when a dir has been created, and then leave out the return otherwise (so `undefined` is implicitly returned)